27_11_2019/SimpleImageProcessing/CMakeLists.txt

```diff
@@ -6,5 +6,4 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 add_executable( main main.cpp Matrix.cpp MatrixUtils.cpp Matrix.hpp MatrixUtils.hpp ppImageUtils.hpp ppImageUtils.cpp)
 
-add_custom_command( TARGET main POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/l.pgm  $<TARGET_FILE_DIR:main> )
-add_custom_command( TARGET main POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/l.pgm ${CMAKE_CURRENT_BINARY_DIR} )
+add_custom_command( TARGET main POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/l.pgm ${CMAKE_CURRENT_BINARY_DIR} )
```

Comment:
The one on master works both for linux and windows, so it would be better not to change it.

Reply:
How do I not change it? like can I delete file from my branch and pull the one from master?

Reply:
you can in this case just copy paste the two lines from master to your branch, then commit. More generally you could checkout to master, copy the file, go back to your branch and paste.